src/components/vega-wallet/vega-wallet-not-running.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslation } from "react-i18next";
+import { DEFAULT_WALLET_URL } from "../../lib/vega-wallet/vega-wallet-service";
+import "./vega-wallet-not-running.scss";
+
+export const VegaWalletNotRunning = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="vega-wallet-not-running__container">
+      <div className="vega-wallet-not-running__input">
+        <p>{t("urlLabel")}</p>
+        <input
+          disabled={true}
```

Comment:
Why is this input disabled. If the user has changed their wallet location they wont ever be able to connect

Reply:
There's not interaction here from the user. As soon as the wallet service starts, they won't see this state anymore.

Reply:
This could be a text field instead of a disabled input, but I'm following the design attached to the ticket.

Reply:
I've enabled the text field, and made some changes to listen for url field changes and then update the walletservice url